src/test/java/com/ably/kafka/connect/AblyHelpers.java

```diff
@@ -72,7 +72,7 @@ public String key() {
     public static AppSpec createTestApp() throws AblyException {
         ClientOptions opts = new ClientOptions("a.b:c");
         opts.environment = TEST_ENVIRONMENT;
-        AblyRest ably = new AblyRest(opts);
+        AblyRealtime ably = new AblyRealtime(opts);
```

Comment:
What's the purpose of this change in terms of optimizing tests? :thinking:

Reply:
I found out it was faster to use AblyRealtime + I didn't find reason why we are using AblyRest 🤷